core/src/main/java/io/seata/core/rpc/netty/v1/ProtocolV1Decoder.java

```diff
@@ -74,8 +74,14 @@ public ProtocolV1Decoder(int maxFrameLength) {
     protected Object decode(ChannelHandlerContext ctx, ByteBuf in) throws Exception {
         Object decoded = super.decode(ctx, in);
         if (decoded instanceof ByteBuf) {
-            ByteBuf frame = (ByteBuf) decoded;
-            return decodeFrame(frame);
+            try {
+                ByteBuf frame = (ByteBuf) decoded;
+                return decodeFrame(frame);
+            }catch(Exception e){
+                throw e;
+            }finally {
+                in.release();
```

Comment:
Not release `in` but `frame`

Reply:
agree，`in` can be released in `ByteToMessageDecoder.channelRead`, but `frame` allocate in LengthFieldBasedFrameDecoder.

Reply:
change frame.release and add error logger